src/components/Hero/Hero.style.js

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+
+// eslint-disable-next-line import/prefer-default-export
+export const Wrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 16px;
+`;
+
+export const Title = styled.h2`
+  margin-bottom: 15px;
+  text-align: center;
+  font-weight: 400;
+`;
+
+export const Subtitle = styled.small`
+  color: #93918f;
+  margin-bottom: 30px;
+  font-size: 16px;
+  text-align: center;
+`;
+
+export const Content = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const Small = styled.small`
+  color: #93918f;
+  margin: 26px 0;
+  text-align: center;
+`;
+
+export const CTALink = styled.p`
+  font-size: 14px;
+  color: white;
+  background: #fdb755;
+  text-transform: uppercase;
+  border: none;
+  outline: none;
+  width: 217px;
```

Comment:
Do you remember why this width was required? My guess would be that it can be removed

Reply:
yea your right, I used padding instead 👍